doc/api/stream.markdown

```diff
@@ -963,19 +961,16 @@ becomes available.  There is no need, for example to "wait" until
   Buffer encoding, such as `'utf8'` or `'ascii'`
 * return {Boolean} Whether or not more pushes should be performed
 
-Note: **This function should be called by Readable implementors, NOT
+Note: **This method should be called by Readable implementors, NOT
 by consumers of Readable streams.**
 
-The `_read()` function will not be called again until at least one
-`push(chunk)` call is made.
-
-The `Readable` class works by putting data into a read queue to be
-pulled out later by calling the `read()` method when the `'readable'`
-event fires.
+If a String or Buffer is passed, The `push()` method adds a chunk of data 
```

Comment:
This gets a little into the weeds, but this should be "if a non-null value is passed, the `push()` method adds it into the queue for subsequent..." (taking into account `objectMode`.)